exceptions.txt

```diff
@@ -1,31 +1,26 @@
-72546: 'CSI',
-73696: 'CSI: New York',
 110381: 'Archer',
 83897: 'Life After People: The Series',
-80552: 'Kitchen Nightmares (US)', 'Kitchen Nightmares US',
+80552: 'Kitchen Nightmares US',
 75692: 'Law & Order: SVU',
 71489: 'Law and Order CI', 'Law & Order: CI',
 79590: 'Dancing With The Stars (US)',
-73387: 'The Late Late Show with Craig Ferguson', 'Craig Ferguson',
+73387: 'Craig Ferguson',
 75088: 'David Letterman',
 76706: 'Big Brother (US)',
-105521: 'The Colony', 'The Colony (US)',
-76235: 'America\'s Funniest Home Videos', 'AFHV',
-139941: 'Childrens Hospital (US)', 'Childrens Hospital',
-83123: 'Merlin', 'Merlin (2008)',
-76779: 'WWE Monday Night RAW',
-164951: 'Shit My Dad Says',
-168161: 'Law & Order: Los Angeles', 'Law & Order: LA',
+105521: 'The Colony (US)',
+76235: 'AFHV',
+139941: 'Childrens Hospital (US)',
+83123: 'Merlin (2008)',
+168161: 'Law & Order: LA',
 77526: 'Star Trek: TOS',
-72194: 'The Ellen Degeneres Show', 'Ellen Degeneres',
+72194: 'Ellen Degeneres',
 72073: 'Star Trek DS9',
-195831: 'Zane Lamprey\'s Drinking Made Easy',
-76133: 'Poirot', 'Agatha Christie\'s Poirot', 'Agatha Christie Poirot',
+76133: 'Agatha Christie Poirot',
 77444: 'This Old House Program',
 73290: '60 Minutes (US)',
-194751: 'Conan', 'Conan (2010)',
+194751: 'Conan',
 164451: 'Carlos (2010)',
-70726: 'Babylon 5', 'Babylon5',
```

Comment:
@miigotu The "Babylon 5" is required? Why is that, since it's identical to TheTVDB name?